### ISSUE TITLE
Can't get all folder

### ISSUE THREAD

**Issue by ghost** [Garethp/php-ews#107]
hey, i am trying to get these folder all of them but can't seem to get even one name and id of that folder.

	$api = MailAPI::withUsernameAndPassword($host, $username, $password);
	$folder1 = $api->getFolderByDisplayName('inboxfolder', DistinguishedFolderIdNameType::INBOX);
	echo $folder1->getFolder();

this is the code. I want to get the folder name and id of all as seen in picture. Hoping to hearing from you soon. Cheers!
![capture](https://user-images.githubusercontent.com/18285618/31314666-ac5466e8-ac1f-11e7-93ef-2e63347384fe.PNG)
Note am using office 365

**Comment by Garethp** [comment]
What do you get when you try

```php
$api = MailAPI::withUsernameAndPassword($host, $username, $password);
$folder1 = $api->getFolderByDisplayName('inboxfolder', DistinguishedFolderIdNameType::INBOX);
var_dump($folder1->getFolder());
```

**Comment by Robinouh** [comment]
Hi,

I have the same problem.

I tried the solution above but I got an error 500.

I would like to know if I can get all folders of the mailbox ?

![sans titre](https://user-images.githubusercontent.com/35724895/36248865-2a17832e-1239-11e8-8170-c1c64134d7e8.png)

For example, as shown in the image above, the "Boîte de réception", "Boîte d'envoi", "Brouillons", "Courriers indésirables", "Éléments envoyés" and "Éléments supprimés" folders should be retrieved.

Is it possible?
